isis/src/base/apps/phocube/main.cpp

```diff
@@ -532,17 +514,19 @@ void phocube(Buffer &out) {
 
       // Trim outer space except RA and dec bands
       else {
-        for (int b = (skipDN) ? 1 : 0; b < nbands; b++) {
-          if(ra && b == raBandNum) {
+        for (int band = (dn) ? 1 : 0; band < nbands; band++) {
+          if(ra && band == raBandNum) {
             out[index] = cam->RightAscension();
+            index += 64 * 64;
           }
-          else if (declination && b == raBandNum + 1) {
+          else if (declination && band == raBandNum + 1) {
             out[index] = cam->Declination();
+            index += 64 * 64;
           }
           else {
             out[index] = Isis::NULL8;
+            index += 64 * 64;
           }
-          index += 64 * 64;
```

Comment:
Why did this move inside the if/if-else/else block? This is the same but it's more confusing now.

Reply:
I originally wrote it that way because I thought it made the code more compact (in a previous change I worked on about a year ago). I then decided to move the line into the if/else block because I thought it increased readability. I can change it back if the cons outweigh the pros.